src/configuration.cpp

```diff
@@ -193,12 +193,14 @@ ssr::conf_struct ssr::configuration(int& argc, char* argv[])
   // load system-wide config file (Linux et al.)
   load_config_file("/etc/ssr.conf",conf);
   // load user config file (Mac)
-  std::string filename = getenv("HOME");
+  std::string filename = pathtools::get_home_dir().string();
   filename += "/Library/SoundScapeRenderer/ssr.conf";
+  filename = pathtools::normalize_path(filename);
   load_config_file(filename.c_str(),conf);
```

Comment:
`filename` could still be a `path` here, which also provides a `.c_str()`  method.